rules-reviewed/eap6/uncategorized/environment-dependent.windup.xml

```diff
@@ -23,7 +23,7 @@
                 </javaclass>
             </when>
             <perform>
-                 <hint title="Dynamic class instantiation" effort="0" category-id="optional">
+                 <hint title="Dynamic class instantiation" effort="1" category-id="optional">
```

Comment:
@d-s  I agree that it has an impact on aggregated efforts. 

Thanks for the hint on the override feature. I will then override the classloading rule from my side and we should discuss the effort aggregation later.